ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -1020,6 +1020,51 @@ abstract class QueryStoreAndAuthDependentIntegrationTest
         }
       }
     }
+
+    "containing a decimal " - {
```

Comment:
This test fails for both in-memory and database lookups without the `apiValueToLfValue` change.